tests/Cluster/ClusterCommTest.cpp

```diff
@@ -197,7 +197,7 @@ TEST_CASE("ClusterComm::wait", "[cluster][mev]") {
     std::future<void> f1(std::async(std::launch::async, [&]{
           timespec ts={0,15000000};
 #if defined(_WIN32)
-          usleep(ts.tv_nsec / 1000L);
+          std::this_thread::sleep_for(std::chrono::microseconds(ts.tv_nsec / 1000L));
 #else
           nanosleep(&ts, nullptr);
```

Comment:
I think nanosleep should be done using sleep_for as well? also the ifdef can go away then

Reply:
Yes, will remove that.